virtual_feature.pl

```diff
@@ -3507,4 +3507,29 @@ sub feature_delete_protected_dir
 return wantarray ? ($err, $status) : $err;
 }
 
+# feature_sysinfo()
+# Returns Nginx and available PHP version
+sub feature_sysinfo
+{
+my @rv = ( [ $text{'sysinfo_nginx'}, &get_nginx_version() ] );
+my @avail = &virtual_server::list_available_php_versions();
+my @vers;
+foreach my $a (grep { $_->[1] } @avail) {
+	&clean_environment();
+	my $out = &backquote_command("$a->[1] -v 2>&1 </dev/null");
```

Comment:
There's a method `get_php_version` which will get the version from a PHP executable

Reply:
Thanks! Fixed now!